### PR TITLE
Fix for Tenant List Dropdown Feature Pushed Parameters

### PR DESCRIPTION
**Pull request by thariyarox**
Fix for Tenant List Dropdown Feature Pushed Parameters

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/tenant_refresh_endpoint.jsp [hunk 1]
```diff
@@ -22,14 +22,14 @@
...
+    private static final String IS_TENANT_LIST_MODIFIED = "isTenantListModified";
 %>
 
 <%
-    String tenantList = request.getParameter(TENANT_LIST);
+    String isTenantListModified = request.getParameter(IS_TENANT_LIST_MODIFIED);
 
-    if (StringUtils.isNotEmpty(tenantList)) {
-        TenantDataManager.setTenantDataList(tenantList);
+    if (StringUtils.isNotEmpty(isTenantListModified) && Boolean.parseBoolean(isTenantListModified)) {
```
Comment (cdwijayarathna) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/tenant_refresh_endpoint.jsp]: Unnecessary null check, please fix.

**Comment by cdwijayarathna** [comment]
Reviewed and verified

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/TenantDataManager.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2014-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (omindu) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/TenantDataManager.java]: Check license header. Should be 2014.

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/listener/AuthenticationEndpointTenantActivityListener.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2013-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (omindu) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/listener/AuthenticationEndpointTenantActivityListener.java]: Check license header.

**Comment by cdwijayarathna** [comment]
Thanks @thariyarox , merged!

**Pull request merged**